qt/scientific_interfaces/ISISReflectometry/GUI/Runs/ISearcher.h

```diff
@@ -18,14 +18,15 @@ namespace CustomInterfaces {
 class SearcherSubscriber {
 public:
   virtual void notifySearchComplete() = 0;
+  virtual void notifySearchFailed() = 0;
 };
 
 /** @class ISearcher
 
 ISearcher is an interface for search implementations used by
 IRunsPresenter implementations.
 */
-class ISearcher {
+class ISearcher : public QObject {
```

Comment:
Could the inheritance of QObject be in the `CatalogSearcher` rather than this interface?

Reply:
I believe since I am connecting a ISearcher object this needs the QObject

Reply:
Isn't the connection being done inside CatalogSearcher? Anyway, it still seems to work if I move it!

Reply:
I seem to be getting this error when compiling if I move it:
![image](https://user-images.githubusercontent.com/24723609/60511197-4a466e00-9cc9-11e9-8dcf-64be0c5235c4.png)